src/cozmo/action.py

```diff
@@ -146,6 +148,16 @@ def is_completed(self):
         return self._state in (ACTION_SUCCEEDED, ACTION_FAILED)
 
     @property
+    def succeeded(self):
```

Comment:
should be `is_succeeded` or perhaps `has_succeeded` to match the pattern of the other boolean properties